### PR TITLE
created separate script for clonning theme

### PR DESCRIPTION
**Pull request by m-ali-0001**
### Description

[LP-](https://philanthropyu.atlassian.net/browse/LP-)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/###)

Add a description of your changes here.

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: scripts/jenkins-common.sh [hunk 1]
```diff
@@ -76,13 +76,4 @@ SHARD_NUM=${SHARD:="all"}
...
-
-theme_clone_path='/edx/src/philu-edx-theme'
-if [ -e $theme_clone_path ]; then
-    rm -rf $theme_clone_path
-fi
-sudo mkdir -p $theme_clone_path;
-sudo chmod 755 -R /edx/src;
-sudo chown -R jenkins:jenkins /edx/src;
-git clone https://philanthropyu:${THEME_USER_PASSWORD}@github.com/philanthropy-u/philu-edx-theme.git $theme_clone_path
+echo "This node is `curl http://169.254.169.254/latest/meta-data/hostname`"
```
Comment (emraan004) [inline scripts/jenkins-common.sh:79]: Please EOF line here

**Review by emraan004 — CHANGES_REQUESTED** [review]

**Review by emraan004 — APPROVED** [review]
LGTM 👍

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**